inc/namespace.php

```diff
@@ -2,12 +2,13 @@
 
 namespace Altis\Security;
 
+use Altis\Security\PHP_Basic_Auth;
 use const Altis\ROOT_DIR;
 use function Altis\get_config;
-use function Altis\get_environment_type;
 
 function bootstrap() {
-	add_action( 'plugins_loaded', __NAMESPACE__ . '\\on_plugins_loaded', 1 );
+	add_action( 'plugins_loaded', __NAMESPACE__ . '\\on_plugins_loaded' );
+	add_action( 'init', __NAMESPACE__ . '\\on_init', 1 );
```

Comment:
If your plugin needs to load early can you use the `muplugins_loaded` action instead perhaps?